Testing/functional/tests/cypress/integration/e2e/dependent/report.js

```diff
@@ -0,0 +1,446 @@
+import {
+    getCardSelector,
+    getTabButtonSelector,
+} from "../../../support/functions/dependent";
+
+const existingDependent = {
+    hdid: "162346565465464564565463257",
+    phn: "9874307175",
+    dateOfBirth: "2015-Aug-20",
+    otherDelegateCount: 0,
+};
+
+describe("Reports", () => {
+    let sensitiveDocText =
+        " The file that you are downloading contains personal information. If you are on a public computer, please ensure that the file is deleted before you log off. ";
+    beforeEach(() => {
+        cy.setupDownloads();
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+                timelineEnabled: true,
+                datasets: [
+                    {
+                        name: "note",
+                        enabled: false,
+                    },
+                ],
+            },
+            datasets: [
+                {
+                    name: "clinicalDocument",
+                    enabled: true,
+                },
+                {
+                    name: "covid19TestResult",
+                    enabled: true,
+                },
+                {
+                    name: "healthVisit",
+                    enabled: true,
+                },
+                {
+                    name: "hospitalVisit",
+                    enabled: true,
+                },
+                {
+                    name: "immunization",
+                    enabled: true,
+                },
+                {
+                    name: "labResult",
+                    enabled: true,
+                },
+                {
+                    name: "medication",
+                    enabled: true,
+                },
+                {
+                    name: "note",
+                    enabled: true,
+                },
+                {
+                    name: "specialAuthorityRequest",
+                    enabled: true,
+                },
+            ],
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/dependents"
+        );
+    });
+
+    it("Validate Service Selection", () => {
+        const hdid = existingDependent.hdid;
+
+        const cardSelector = getCardSelector(hdid);
+        const tabButtonSelector = getTabButtonSelector(hdid, "report");
+        const tabSelector = `${cardSelector} [data-testid=report-tab]`;
+
+        cy.get(tabButtonSelector)
+            .should("be.visible")
+            .should("not.be.disabled")
+            .should("not.have.class", "disabled")
+            .click();
+
+        cy.get(`${tabSelector} [data-testid=exportRecordBtn] button`).should(
+            "be.disabled",
+            "be.visible"
+        );
+
+        cy.get(`${tabSelector} [data-testid=infoText]`).should(
+            "have.text",
+            " Select a record type above to create a report "
+        );
```

Comment:
We don't need to check the text content (it's static); we can just check if the element is visible.